exercisePPJ_06_I/FunctionPPJ_06_I.java

```diff
@@ -0,0 +1,52 @@
+public class FunctionPPJ_06_I {
+	
+	private double binaryWrt[] = {0,0,0,0,0,0,0,0};
+	private double wrtDouble = 0;
+	
+	public void setFunction(double wrtD) {
+		wrtDouble = wrtD;
+	}
+	
+	public void getFunction() {
+		
+		for(int counter = 7; counter >= 0; counter--) {
+		
+		int exponent = 0;
+		
+		while(wrtDouble > Math.pow(2,exponent)) {
+			exponent++;
+		}
+		
+		if(wrtDouble == Math.pow(2,exponent)) {
+			binaryWrt[exponent] = Math.pow(2,exponent);
+		}
+		else {
+			binaryWrt[exponent-1] = Math.pow(2,(exponent-1));
+			
+			int highestExponent = exponent-1;
+			double rest = wrtDouble - Math.pow(2,highestExponent);
+			
+			wrtDouble = rest;
+			exponent=0;
+		}
+	}
+
+		}
+	public void showFunction() {
+		for(int exponent = 0; exponent <= 7; exponent++) {
+		System.out.println("2^"+exponent+": "+binaryWrt[exponent]);
```

Comment:
In such case it might be a good idea to use `printf` instead of `println`
It's quite good to use `printf` all the time, because it's worth to learn.